### PR TITLE
implemented doubly linked list

### PR DESCRIPTION
**Pull request by arielparcon**
implemented doubly linked list in forwards and backwards.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,56 +1,25 @@
...
-
-    // Display the linked list
-    fmt.Println("Linked List:")
-    myList.Display()
+	// Create a new linked list
+	myList := models.DoublyLinkedList{}
+
+	// elements to the linked list
+	l := []int{1,2,3,4,5}
+	myList.AddNode(1)
```
Comment (rexposadas) [inline main.go]: Nope. The signature of the function needs to change. 

this line should be: myList.AddNodes(l) // that's a lower case L

**Review by rexposadas — CHANGES_REQUESTED** [review]

File: models/doubly-linked-list.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	prev *Node // A pointer to the previous node in the list
+}
+
+// DoublyLinkedList represents a doubly linked list
+type DoublyLinkedList struct {
+	head *Node // A pointer to the first node (head) of the linked list.
+}
+
+// AddNode adds a new node to the end of the linked list
+func (list *DoublyLinkedList) AddNode(data int) {
```
Comment (rexposadas) [inline models/doubly-linked-list.go]: Change the function signature to :   AddNodes(data []int)

**Comment by rexposadas** [comment]
@tanggol69  Use underscores in your file names. instead of `doubly-linked-list.go` it should be `doubly_linked_list.go`.  I believe that's standard Unix convention of naming a file.

File: main.go (hunk 1 above)
Comment (arielparcon) [reply main.go]: changed the signature of the function.

File: models/doubly_linked_list.go [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+	prev *Node // A pointer to the previous node in the list
+}
+
+// DoublyLinkedList represents a doubly linked list
+type DoublyLinkedList struct {
+	head *Node // A pointer to the first node (head) of the linked list.
+}
+
+// AddNode adds a new node to the end of the linked list
+func (list *DoublyLinkedList) AddNodes(data int) {
```
Comment (rexposadas) [inline models/doubly_linked_list.go]: You need to change the signature of this method to : 

AddNotes(data []int)

**Review by rexposadas — CHANGES_REQUESTED** [review]

File: models/doubly-linked-list.go (hunk 2 above)
Comment (arielparcon) [reply models/doubly-linked-list.go]: requested changes addressed.

File: models/doubly_linked_list.go (hunk 3 above)
Comment (arielparcon) [reply models/doubly_linked_list.go]: requested changes addressed.

**Review by rexposadas — APPROVED** [review]

**Pull request merged**